nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/main/java/org/apache/nifi/groups/StandardProcessGroup.java

```diff
@@ -2168,6 +2170,16 @@ private boolean isDisconnected(final Snippet snippet) {
         return true;
     }
 
+    @Override
+    public Set<Positionable> findAllPositionables() {
+        Set<Positionable> positionables = Sets.newHashSet();
+        positionables.addAll(findAllConnectables(this, true));
+        positionables.addAll(findAllProcessGroups());
```

Comment:
We should check to see if this recurses within nested process groups when scaling top-level process group.